rdrf/rdrf/templates/registration/email_reset_activation.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+<p>
+  {% trans "Hello" %} {{user_full_name}},<br />
```

Comment:
Should use `blocktrans` so that translators can re-arrange templated values

Reply:
Fixed